_____new-technologies-to-integrate/genkit.md

```
@@ -0,0 +1,31 @@
+docs are here:
+
+https://console.firebase.google.com/u/0/project/jeoparody-e73a2/genai_monitoring
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)